PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -28,6 +28,7 @@ def miniAOD_customizeCommon(process):
     process.patMuons.computeMiniIso = cms.bool(True)
     process.patMuons.computeMuonMVA = cms.bool(True)
     process.patMuons.computeSoftMuonMVA = cms.bool(True)
+    process.patMuons.computePuppiCombinedIso = cms.bool(True)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
@slava77  done! #26233